doc/readme/DEVELOPING.md

```diff
@@ -1,7 +1,7 @@
 # Developing Shogun
 
-This is a very basic list of things how to get started hacking Shogun.
-Your first steps should be to
+This is a very basic checklist on how to get started with Shogun,
+You should:
```

Comment:
Maybe just drop the `you should:`, i.e. `This is a very basic checklist on how to get started with Shogun:`

Reply:
great, will do